frontend/src/pages/books/BookList.tsx

```diff
@@ -61,6 +62,8 @@ export interface Book {
   daysOfSupply?: string | number;
   lineItems?: BookDetailLineItem[];
   isGhost?: boolean;
+  numRelatedBooks?: number;
```

Comment:
Why is it optional? Isn't it provided with all versions of the book?

Reply:
Same question with relatedBooks, I feel like both of these shouldn't be optional. The ones above this are optional because they are either calculated or not provided with all versions of the book

Reply:
Due to the internal to APIBook conversion code using Book and APIBook in the books conversion file, I didn't want to actually define a relationship in sending that data back so I left it as optional. Does that make sense or should I just go ahead and do it instead?

Reply:
Ahh I see, yeah it's more of an issue with how we have the code structured. It's fine for now